tools/cpp/unix_cc_configure.bzl

```diff
@@ -273,7 +273,14 @@ def _is_clang(repository_ctx, cc):
 def _is_gcc(repository_ctx, cc):
     # GCC's version output uses the basename of argv[0] as the program name:
     # https://gcc.gnu.org/git/?p=gcc.git;a=blob;f=gcc/gcc.cc;h=158461167951c1b9540322fb19be6a89d6da07fc;hb=HEAD#l8728
-    return repository_ctx.execute([cc, "--version"]).stdout.startswith("gcc ")
+    cc_argv0 = repository_ctx.execute([cc, "--version"]).stdout.partition(" ")[0]
+    if (cc_argv0 == "gcc"):
+        return True
+
+    if (cc_argv0.startswith("gcc-")):
+        return cc_argv0.removeprefix("gcc-").isdigit()
```

Comment:
I think that starting with `gcc-` is probably good enough of an indicator without a separate check of the characters that follow. In fact, an executable such as `gcc-arm-none-eabi` should ideally also be recognized as `gcc`.

Reply:
Dropped the `isdigit()`-check!